packages/jsActions/native-mobile-resources/src/camera/TakePicture.ts

```diff
@@ -127,17 +128,27 @@ export async function TakePicture(
     // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
     function getOptions() {
         const { maxWidth, maxHeight } = getPictureQuality();
+        const [language] = getLocales().map(local => local.languageCode);
+        const isDutch = language === "nl";
 
         return {
             mediaType: "photo" as "photo",
             maxWidth,
             maxHeight,
             noData: true,
+            title: isDutch ? "Foto toevoegen" : "Select a photo",
+            cancelButtonTitle: isDutch ? "Annuleren" : "Cancel",
+            takePhotoButtonTitle: isDutch ? "Foto maken" : "Take photo",
+            chooseFromLibraryButtonTitle: isDutch ? "Kies uit bibliotheek" : "Choose from library",
             permissionDenied: {
-                title: "This app does not have access to your camera or photos",
-                text: "To enable access, tap Settings > Permissions and turn on Camera and Storage.",
-                reTryTitle: "Settings",
-                okTitle: "Cancel"
+                title: isDutch
+                    ? "Dokter Dichtbij heeft geen toegang tot uw foto's"
```

Comment:
Deze app heeft geen toegang tot uw camera of bestanden.

Reply:
There's still Dokter Dichtbij in the texts! :-)

Reply:
You are seeing an old commit @mgroeneweg